src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.tsx

```diff
@@ -229,6 +238,26 @@ const StatusForm: FunctionComponent<StatusFormProps> = ({
     }
   }, [escFunction, listenFor, unlisten])
 
+  useEffect(() => {
+    if (!emailTemplate) return
+
+    if (emailTemplate?.html) {
+      openEmailPreviewModal()
+      dispatch({ type: 'SET_EMAIL_TEMPLATE', payload: emailTemplate })
+    }
+  }, [emailTemplate, openEmailPreviewModal, dispatch])
+
+  useEffect(() => {
+    if (emailTemplateError) {
+      dispatch({
+        type: 'SET_ERRORS',
+        payload: {
+          text: 'Er is geen email template beschikbaar voor de gegeven status transitie',
+        },
+      })
+    }
```

Comment:
Does this mean that, whenever there is an error on the API side, the changes aren't saved? Not sure if that is very user friendly.

Reply:
changes are saved now